### PR TITLE
Bump MSRV to 1.60

### PR DESCRIPTION
**Pull request by h3r2tic**
Needed for `bytemuck`: https://github.com/Lokathor/bytemuck/pull/140#issuecomment-1304404474

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,103 +1,103 @@
-# 🎥 dolly
```
Comment (MarijnS95) [inline README.md]: I'm sure that you didn't want to change this entire file to CRLF!

Comment (h3r2tic) [reply README.md]: WHAT IN THE... ARGH. Thanks.

Comment (MarijnS95) [reply README.md]: Just Windows things™? :grimacing:

Comment (h3r2tic) [reply README.md]: Yup... Had autocrlf disabled due to some recent issues when running shell scripts from a Linux container inside a repo synced on Windows... CRLF is a gift that keeps on giving.

Comment (h3r2tic) [reply README.md]: Phew, fixed. Using the classic method of "screw everything, re-download the file from `main`, and apply fixes on top".

Comment (MarijnS95) [reply README.md]: `dos2unix`? Works every time :stuck_out_tongue:

Comment (h3r2tic) [reply README.md]: 😲 <- mfw I realize I have it in Windows.

Comment (MarijnS95) [reply README.md]: MSYS2?

Comment (h3r2tic) [reply README.md]: git 👀

Comment (MarijnS95) [reply README.md]: git-for-windows is effectively MSYS2 with changes / preincluded packages / optimized `git` package specifically for Windows.

Comment (h3r2tic) [reply README.md]: image

Comment (MarijnS95) [reply README.md]: :joy: :joy: :joy: :joy:

**Pull request merged**